config.cfg

```diff
@@ -19,6 +19,10 @@ modelfile = missing
 # Enable the logfile (1) or disable the logfile (0).
 enablelogfile = 1
```

Comment:
I suggest to remove this configuration option and handle logging as follows:
- Write all logs to stdout and stderr
- Log statement include a log level such as debug, info, warn or error
- The logging module is configured to write warn/error to stderr
- The logging module is configured to write info to stdout
- The logging module is configured to write debug to stdout when the user enabled verbose mode using -v --verbose
- Do not include timestamps in log statement, but configure the logging module to prepend statements with timestamps

The behavior then becomes standard to that of many Linux command-line applications and ensures smooth integration of CAPICE within VIP. Furthermore it prevents the read-only error when running CAPICE in a Singularity container.

Reply:
Fixed in #45